tests/zap/test_access_and_admin_functions.py

```diff
@@ -135,12 +119,90 @@ def test_execute_non_owner(alice, union_contract):
 
 def test_claim_non_owner(alice, union_contract):
     with brownie.reverts("Ownable: caller is not the owner"):
-        union_contract.claim([(ADDRESS_ZERO, 0, 0, ["0x0"])], {"from": alice})
+        union_contract.claim(DUMMY_PROOF, {"from": alice})
+
+
+def test_claim_no_claims(owner, union_contract):
+    with brownie.reverts("No claims"):
+        union_contract.claim([], {"from": owner})
+
+
+def test_swap_non_owner(alice, union_contract):
+    with brownie.reverts("Ownable: caller is not the owner"):
+        union_contract.swap(DUMMY_PROOF, 0, False, 0, 0, [10000, 0, 0], {"from": alice})
+
+
+def test_swap_invalid_weights(owner, union_contract):
+    with brownie.reverts("Invalid weights"):
```

Comment:
This test shouldn't pass since the error msg is different - `swap` doesn't have the `validWeights` modifier

Reply:
yeah haven't got to updating the tests yet. will do.